css/search.css

```diff
@@ -0,0 +1,96 @@
+   * {
+  box-sizing: border-box;
+}
+.search {
+  width: 100px;
+  height: 100px;
+  margin: 40px auto 0;
+  background-color: #00FEDE;
+  position: relative;
+  overflow: hidden;
+  transition: all 0.5s ease;
+  border-radius: 70px;
```

Comment:
Use 50% border-radius to make things round, instead of some px value, with 50% it will always be a perfect circle does not meter how big the container is